blog/electron-17-0.md

```diff
@@ -0,0 +1,81 @@
+      url: 'https://github.com/VerteDinde'
+      image_url: 'https://github.com/VerteDinde.png?size=96'
+slug: electron-17-0
+
+---
+
+Electron 17.0.0 has been released! It includes upgrades to Chromium `98`, V8 `9.8`, and Node.js `16.13.0`. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 17.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://www.electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
+
+## Notable Changes
+
+### Electron Release Cadence Change
+
+As of Electron 15, Electron will release a new major stable version every 8 weeks. You can read the [full details here](https://www.electronjs.org/blog/8-week-cadence).
+
+Additionally, Electron has changed supported versions from latest three versions to latest four versions until May 2022. [See our versioning document](https://www.electronjs.org/docs/latest/tutorial/electron-versioning) for more detailed information about versioning in Electron. After May 2022, we will return to supporting latest three versions.
+
+### Stack Changes
+
+* Chromium `98`
+    * [New in Chrome 98](https://developer.chrome.com/blog/new-in-chrome-98/)
+* Node.js `16.13.0`
+    * [Node 16.13.0 blog post](https://nodejs.org/en/blog/release/v16.13.0/)
+* V8 `9.8`
+    * [V8 9.8 blog post](https://v8.dev/blog/v8-release-98)
+
+### Highlighted Features
+
+* Added `webContents.getMediaSourceId()`, can be used with `getUserMedia` to get a stream for a WebContents. [#31204](https://github.com/electron/electron/pull/31204)
+* Deprecates `webContents.getPrinters()` and introduces `webContents.getPrintersAsync()`. [#31023](https://github.com/electron/electron/pull/31023)
+* `desktopCapturer.getSources` is now only available in the main process. [#30720](https://github.com/electron/electron/pull/30720)
+
+See the [17.0.0 release notes](https://github.com/electron/electron/releases/tag/v17.0.0) for a full list of new features and changes.
+
+## Breaking Changes
+
+Below are breaking changes introduced in Electron 17. More information about these and future changes can be found on the [Planned Breaking Changes](https://www.electronjs.org/docs/latest/breaking-changes) page.
+
+### desktopCapturer.getSources in the renderer
+
+The desktopCapturer.getSources API is now only available in the main process. This has been changed in order to improve the default security of Electron apps.
+
+### API Changes
+
+There were no API changes in Electron 17. 
+
+### Removed/Deprecated Changes
+
+* Usage of the `desktopCapturer.getSources` API in the renderer has been deprecated and will be removed. This change improves the default security of Electron apps. See [here](https://raw.githubusercontent.com/electron/electron/main/docs/breaking-changes.md#removed-desktopcapturergetsources-in-the-renderer) for details on how to replace this API in your app.
```

Comment:
I'm not sure if there is precedent for this, but it a breaking change involving something removed. I think it should be fine to just keep the mention of it in the `Breaking Changes` section rather than both, unless there are any objections. (It was also in the `Removed/Deprecated` category last time when it was deprecated).

Reply:
I updated the wording to state that is has been removed. After going over it I think it makes sense to keep the structure of the blog posts consistent.